victor/src/main/groovy/com/trello/victor/VictorPlugin.groovy

```diff
@@ -83,15 +84,25 @@ class VictorPlugin implements Plugin<Project> {
                     return
                 }
 
-                Task rasterizationTask = project.task("rasterizeSvgsFor${variant.name.capitalize()}", type: RasterizeTask) {
-                    sources = svgFiles
-                    outputDir = project.file("$project.buildDir/generated/res/$flavorName/$buildType.name/svg/")
-                    includeDensities = densities
-                    baseDpi = project.victor.svgDpi
+
+                File conversionOutputDir = project.file("$project.buildDir/generated/res/$flavorName/$buildType.name/svg/")
```

Comment:
The problem is that the dexer would complain about duplicate resources. So if we would have to clean both folders all the time from the plugin instead of the tasks.